### PR TITLE
[#90] Change types in 'Iris.Colour.Formatting' from 'ByteString' to 'Text'

### LINKED ISSUE #90: Changes types in 'Iris.Colour.Formatting' from 'ByteString' to 'Text'
To make API more consistent. Let's use `Text` everywhere for now.

### PR DESCRIPTION
**Pull request by lillycat332**
Resolves #90

I have changed the ByteString usage in Iris.Colour.Formatting to Data.Text.

### Additional tasks

- [x] Documentation for changes provided/changed
- [ ] Tests added
- [x] Updated CHANGELOG.md

### CODE REVIEW

**Comment by lillycat332** [comment]
We can probably also remove the bytestring dependency if we merge this?

File: CHANGELOG.md [hunk 1]
```diff
@@ -35,6 +34,11 @@ Initial release prepared by [@chshersh](https://github.com/chshersh).
   Add `MonadUnliftIO` instance for the `CliApp` monad
   (by [@charrsky](https://github.com/charrsky))
 
+### Changed
+
+- [#90](https://github.com/chshersh/iris/issues/90):
+  Use Data.Text instead of ByteString for coloured output
```
Comment (chshersh) [inline CHANGELOG.md:40]: Thanks for adding an entry!

Could you move this inside the `## [Unreleased]` section above in the changelog?

**Review by chshersh — CHANGES_REQUESTED** [review]
Thanks, looks great! Nice and simple 🙂 

I have one comment about the changelog entry but the rest looks great 👍🏻

File: src/Iris/Colour/Formatting.hs [hunk 2]
```diff
@@ -20,16 +20,16 @@ module Iris.Colour.Formatting
 
 import Control.Monad.IO.Class (MonadIO (..))
 import Control.Monad.Reader (MonadReader)
-import Data.ByteString (ByteString)
+import qualified Data.Text as T
```
Comment (chshersh) [inline src/Iris/Colour/Formatting.hs:23]: GHC outputs the warning that this import is redundant. I believe, only `IO` functions from `Text` are used so this one can be removed 🧹

**Review by chshersh — COMMENTED** [review]
Everything looks great 👍🏻 

I see a GHC warning about unused import. I believe, it's the only remaining part 🙂

File: src/Iris/Colour/Formatting.hs (hunk 2 above)
Comment (lillycat332) [reply src/Iris/Colour/Formatting.hs]: Removed the redundant import, though it now won't compile though as the simple-grep uses ByteString still.

**Review by chshersh — APPROVED** [review]
Looks fine then 👌🏻 

@lillycat332 Could you resolve conflicts so I can merge the PR? Thanks!

**Comment by lillycat332** [comment]
Done 🙂

**Pull request closed without merging**